analysis/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Byte-compiled
+__pycache__/
+
+# Environments
+.venv
+venv/
+
+# PyCharm files
+.idea
+idea/
+../.idea/
```

Comment:
According to https://www.notion.so/serokell/gitignore-3d45b853c4494fa282dc04269f851b2e, files created by your tools should be `.gitignore`'d globally for each person using them instead of `.gitignore`'ing them in each repo.

Reply:
Configured my global `.gitignore` and deleted this local one, thank you!